### PR TITLE
Feature/data service

### PR DESCRIPTION
**Pull request by Mnw2212**

### CODE REVIEW

File: src/app/components/data/connectdb.service.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    this.fa = $firebaseAuth;
+    this.firebaseURL = "https://vyapi.firebaseio.com";
+    this.ref = null
+    this.authObj = null
+    this.authData = this.authObj.$getAuth();
+    this.clog = $log;
+  }
+
+  connect(path='/'){
+    this.ref = new Firebase("https://vyapi.firebaseio.com"+path);
```
Comment (skeep) [inline src/app/components/data/connectdb.service.js]: use `template string`

File: src/app/components/data/connectdb.service.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    this.authObj = this.fa(this.ref);
+    this.clog.log("authObject returned, connection made to "+path);
```
Comment (skeep) [inline src/app/components/data/connectdb.service.js]: use `.info` instead of `.log`

File: src/app/components/data/data.service.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      return false;
+    }
+  }
+
+  getUid(){
+    var authDataLocal = authExist();
+    return authDataLocal.uid;
+  }
+
+  getRooms(userID){
```
Comment (skeep) [inline src/app/components/data/data.service.js]: these should mov to `room.service.js`

File: src/app/components/data/data.service.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    var ref = this.fc.connect(this.roomPath);
+    return ref.orderByChild("ownedBy").equalTo(userID);
+  }
+
+  createRoom(userID){
```
Comment (skeep) [inline src/app/components/data/data.service.js]: this is job of room controller

**Comment by Mnw2212** [comment]
@skeep Rectified the mistakes

**Pull request merged**